.circleci/config.yml

```diff
@@ -2,15 +2,33 @@ version: 2.1
 
 orbs:
   ft-golang-ci: financial-times/golang-ci@1
+  docker: circleci/docker@1.7.0
+
+jobs:
+  build-and-publish-docker-image:
+          docker:
+            - image: docker:stable-git
+          steps:
+            - checkout
+            - setup_remote_docker
+            - run:
+                name: Build Docker image
+                command: docker build --tag coco/publish-failiure-resolver-go:latest .
```

Comment:
This should be `publish-failure-resolver-go` for it to push to the right dockerhub repo -> https://hub.docker.com/repository/docker/coco/publish-failure-resolver-go

Reply:
The typo is fixed. Thanks for the review.